controllers/configs.go

```diff
@@ -89,7 +90,24 @@ func (r *TalosControlPlaneReconciler) talosconfigForMachines(ctx context.Context
 
 	addrList := []string{}
 
-	var t *talosconfig.Config
+	var (
+		talosconfigSecret v1.Secret
+	)
+
+	if err := r.Client.Get(ctx,
+		types.NamespacedName{
+			Namespace: "default",
```

Comment:
is this always the `default` namespace?

Reply:
nice catch, it should be Machine's namespace (or Cluster's)